src/tapeimage.cpp

```diff
@@ -27,6 +27,7 @@ class tapeimage : public lfp_protocol {
 
     void seek(std::int64_t)   noexcept (false) override;
     std::int64_t tell() const noexcept (false) override;
+    lfp_protocol* peel() noexcept (false) override;
```

Comment:
Why is this noexcept(false)? I don't think anything can throw here.

Reply:
It can't in the case of tapeimage. But it needs to match the the virtual function it overrides, which is noexcept(false) in order to throw not implemented. It's the same story as for `tell()`really.